src/app/three-examples/objects/abstract-mesh-3d.ts

```diff
@@ -0,0 +1,32 @@
+import { Input } from '@angular/core';
+import * as THREE from 'three';
+import { AbstractObject3D } from './abstract-object-3d';
+
+export abstract class AbstractMesh extends AbstractObject3D<THREE.Mesh> {
+  @Input()
+  material: string;
+  @Input()
+  materialcolor: number;
+
+  constructor() { 
+    super();
+    console.log('AbstractMesh.constructor');
+  }
+
+
+  public getMaterial(): THREE.MeshMaterialType {
+    let rmaterial: THREE.MeshMaterialType;
+    let appliedColor: number = 0xffff00; 
```

Comment:
Unnecessary type, TSLint should have told you. Please run TSLint after you are through all review comments.
```suggestion
    let appliedColor = 0xffff00; 
```